airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/scheduling/GeographyMapper.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.temporal.scheduling;
```

Comment:
This interface is in the `airbyte-commons-temporal` module, so that on the Cloud side, the new `cloud-workers` module can depend on `airbyte-commons-temporal` without needing a dependency on `airbyte-workers`. 

I debated putting this interface in `airbyte-commons-worker` but based on the contents of the two modules, `airbyte-commons-temporal` seemed like a slightly better fit. Definitely open to other opinions here!